rosbridge_library/src/rosbridge_library/internal/message_conversion.py

```diff
@@ -127,29 +144,32 @@ def configure(node_handle=None):
             sys.exit(0)
 
 
-def get_encoder():
+def get_encoder() -> Callable[[ListType], bytes]:
     configure()
+    assert binary_encoder is not None, "Binary encoder is not configured"
     return binary_encoder
 
 
 class InvalidMessageException(Exception):
-    def __init__(self, inst):
+    def __init__(self, inst: object) -> None:
```

Comment:
In my opinion object doesn't have a place anymore in python since python 3. I think `Any` would be more suitable here.

Reply:
Due to [ANN401](https://docs.astral.sh/ruff/rules/any-type/) check, we can't use unconstrained type for function arguments